### PR TITLE
Adding vulnerability fixes

### PR DESCRIPTION
**Pull request by sadasivaprasad-trellix**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -108,10 +112,10 @@ dependencies {
     // http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2019-16942
     // http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2019-16943
     // CVE-2020-25649
-    compile ('com.fasterxml.jackson.core:jackson-databind:2.13.2.2') {
+    compile ('com.fasterxml.jackson.core:jackson-databind:2.16.0') {
```
Comment (wgibbo-trellix) [inline build.gradle:115]:
```suggestion
    compile ('com.fasterxml.jackson.core:jackson-databind:2.16.1') {
```

File: build.gradle [hunk 2]
```diff
@@ -108,10 +112,10 @@ dependencies {
...
         force = true
     }
-    kafkaInMemory ('com.fasterxml.jackson.core:jackson-databind:2.13.2.2') {
+    kafkaInMemory ('com.fasterxml.jackson.core:jackson-databind:2.16.0') {
```
Comment (wgibbo-trellix) [inline build.gradle:118]:
```suggestion
    kafkaInMemory ('com.fasterxml.jackson.core:jackson-databind:2.16.1') {
```

File: src/main/java/com/opendxl/databus/producer/Producer.java [hunk 3]
```diff
@@ -28,6 +28,8 @@
 
 import java.util.Map;
 import java.util.List;
+import java.time.Duration;
```
Comment (wgibbo-trellix) [inline src/main/java/com/opendxl/databus/producer/Producer.java:31]: Copyright in the file needs updating

File: src/main/java/com/opendxl/databus/common/MessageFormat.java [hunk 4]
```diff
@@ -2,17 +2,19 @@
  * Copyright (c) 2023 Musarubra, LLC - All Rights Reserved.                     *
  *---------------------------------------------------------------------------*/
 
- package com.opendxl.databus.common;
+package com.opendxl.databus.common;
```
Comment (wgibbo-trellix) [inline src/main/java/com/opendxl/databus/common/MessageFormat.java:5]: Copyright in the file needs updating

**Review by wgibbo-trellix — APPROVED** [review]

**Review by gmishra-mcafee — APPROVED** [review]

**Pull request merged**